src/dataServices/dynamoDbBundleServiceHelper.ts

```diff
@@ -24,7 +25,11 @@ export interface ItemRequest {
 }
 
 export default class DynamoDbBundleServiceHelper {
-    static generateStagingRequests(requests: BatchReadWriteRequest[], idToVersionId: Record<string, number>) {
+    static generateStagingRequests(
+        requests: BatchReadWriteRequest[],
+        idToVersionId: Record<string, number>,
+        ttlsInSeconds: Map<string, number>,
```

Comment:
Yeah, I didn't because I defaulted in the persistence and bundle services. However, it did feel weird updating all the helper tests. I'll update to optional and remove from all tests that don't assert the archive logic.

Reply:
pushed an update to make this optional and default. Also, rolled back the tests that were passing in an empty map.